src/routes/get-started/ReportBug.tsx

```diff
@@ -262,18 +262,29 @@ export default function ReportBugPage() {
         <GoabFormItem label="Replication steps" mb="xl" helpText="Detailed steps to reproduce your issue." error={ replicationError }>
           <GoabTextarea name="replication" value={ formValues.replication } onChange={(event: GoabTextAreaOnChangeDetail) => handleChange(event.name, event.value) } rows={ 6 } width="90%" error={ !!replicationError } />
         </GoabFormItem>
-        <GoabFormItem label="StackBlitz URL" mb="s" helpText="Share your code with us in an isolated environment." requirement="optional" error={ stackblitzError }>
+        <GoabFormItem label="StackBlitz URL" mb="s" 
+          helpText={
+            <>Share your code with us in an isolated environment.
+              <GoabBlock mt="xs" gap="m">
+                <a href="https://stackblitz.com/~/github.com/GovAlta/ui-components-react-sandbox">React Stackblitz</a>
+                <a href="https://stackblitz.com/~/github.com/GovAlta/ui-components-angular-sandbox">Angular Stackblitz</a>
+              </GoabBlock>
```

Comment:
I would imagine we would want these as links that load in a new tab...?

Reply:
@chrisolsen @twjeffery This has been changed